pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "pdm.backend"
 
 [project]
 name = "ceph-nvmeof"
-dynamic = ["version"]
+version = "0.0.1"
```

Comment:
@epuertat Looks great overall 👍 I am wondering if the version could be defined in a single place to avoid inconsistency, (since it is defined also in  [.env](https://github.com/ceph/ceph-nvmeof/blob/38ea58f68902beec4a6fd5995a2e6902e0e6e2d3/.env#L2))

Reply:
Is it possible to go other way around and calculate VERSION/NVMEOF_VERSION from pyproject.toml definition?